int64 Status1
int64 Enc1
int64 Crc1
int64 Status2
int64 Enc2
int64 Crc2
